.circleci/config.yml

```diff
@@ -140,17 +140,18 @@ jobs:
       - checkout
       - <<: *fixgit
       - restore_cache:
-          key: deps-v2-dt-{{ checksum "requirements.txt" }}
+          key: deps-v3-dt-{{ checksum "requirements.txt" }}
       - <<: *setup
       - <<: *installdeps
       - save_cache:
-          key: deps-v2-dt-{{ checksum "requirements.txt" }}
+          key: deps-v3-dt-{{ checksum "requirements.txt" }}
           paths:
+            - "~/venv/bin"
```

Comment:
why do you have to add this everywhere

Reply:
So the `coverage` and `pytest` binaries are installed